### ISSUE TITLE
Can not publish episode after upgrade

### ISSUE THREAD

**Issue by KSDaemon** [podlove/podlove-publisher#971]
In my wordpress installation there is also "WP to Twitter" (3.3.2 version) plugin installed.
After upgrading podlove to latest version, episode publishing is broken → ends up in 500 error.

### Expected behavior
Publishing post should really publish post :)

### Actual behavior
episode publishing is broken → ends up in 500 error.
Here is some more info from log:
```
2018/03/29 12:02:54 [error] 21712#21712: *253078 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function podlove_maybe_override_post_titles(), 1 passed in wp-includes/class-wp-hook.php on line 286 and exactly 2 expected in wp-content/plugins/podlove-podcasting-plugin-for-wordpress/includes/auto_post_titles.php:12
Stack trace:
#0 wp-includes/class-wp-hook.php(286): podlove_maybe_override_post_titles('SDCast #75: \xD0\xB2 ...')
#1 wp-includes/plugin.php(203): WP_Hook->apply_filters('SDCast #75: \xD0\xB2 ...', Array)
#2 wp-content/plugins/wp-to-twitter/wp-to-twitter.php(650): apply_filters('the_title', 'SDCast #75: \xD0\xB2 ...')
#3 wp-content/plugins/wp-to-twitter/wp-to-twitter.php(767): wpt_post_info(513)
#4 wp-content/plugins/wp-to-twitter/wp-to-twitter.php(1846): wpt_twee" while reading response header from upstream, request: "POST /wp-admin/post.php HTTP/1.1", upstream: "fastcgi://unix:/var/lib/php/php-fpm.sock:", referrer: "wp-admin/post.php?post=513&action=edit"
```

### System information (see `Podlove > Support` menu)
```
Website                    https://sdcast.ksdaemon.ru
PHP Version                7.2.3
WordPress Version          4.9.4
WordPress Theme            TwentyThirteen SDCast Edition v0.0.1
WordPress Database Charset utf8
WordPress Database Collate 
Publisher Version          2.7.0
Web Player Version         player_v2
Twig Version               1.35.0
open_basedir               ok
curl Version               7.19.7
iconv                      available
simplexml                  ok
max_execution_time         30
upload_max_filesize        2M
memory_limit               256M
disable_classes            
disable_functions          
permalinks                 ok (/%year%/%monthnum%/%postname%/)
podlove_permalinks         ok
podcast_settings           ok
web_player                 ok
podlove_cache              on
assets                     
  - mp3    audio/mpeg       https://sdcast.ksdaemon.ru/feed/podcast/
  - ogg    audio/ogg        no feed
cron                       ok

0 errors
0 notices
Nice, Everything looks fine!
```

**Comment by eteubert** [comment]
I believe this is fixed since 2.7.1 (https://github.com/podlove/podlove-publisher/commit/19c9cd254cda8fbfe12548532df4244828f514b4)

**Comment by KSDaemon** [comment]
Thanks! I'll check it soon!
